setup.py

```diff
@@ -17,7 +17,7 @@
 __author__ = 'Abhinav Singh'
 __author_email__ = 'mailsforabhinav@gmail.com'
 __homepage__ = 'https://github.com/abhinavsingh/proxy.py'
-__download_url__ = '%s/archive/master.zip' % __homepage__
+__download_url__ = '%s/archive/develop.zip' % __homepage__
```

Comment:
How I manage releases currently:

1) Release happen from `master` branch
2) Develop is used for cutting edge version

At some point, I'll create a PR from `develop` -> `master`, which contain all changes between pip releases.

I ideally don't recommend folks to install from `develop`, if they are looking for stability.  Even though develop is tested and I use to everyday locally, it may contain untested code.

Reply:
Ah, I didn't realize. I thought maybe you forgot to update this.